server/app/src/main/java/com/ams/app/util/JwtUtil.java

```diff
@@ -0,0 +1,54 @@
+package com.ams.app.util;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class JwtUtil {
+
+    private String SECRET_KEY = "My$ecre@t";
```

Comment:
Move this to properties file, create a Config Class for properties file and read this from there